packages/amplify-graphql-schema-generator/.npmignore

```diff
@@ -1,5 +1,6 @@
 **/__mocks__/**
 **/__tests__/**
 src
+vpc-db-lambda
```

Comment:
ignoring lambdas sources.
it appears that this package users are consuming `zip` files in `lib` not sources.